phy/plot/_vispy_utils.py

```diff
@@ -276,6 +298,7 @@ def __init__(self, **kwargs):
         self.n_spikes = None
         self._spike_clusters = None
         self._spike_ids = None
+        self._non_empty = False
```

Comment:
actually if you don't mind I'll do this in the next PR (I'll create it as soon as this one is merge), it will be much simpler for me